bucket/vapoursynth.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "R57",
```

Comment:
`57` is sufficient. The `R` prefix is not needed. Please modify checkver and autoupdate accordingly.